lib/cookies.js

```diff
@@ -10,43 +10,28 @@
  * See the License for the specific language governing permissions and limitations under the License.
  *
  */
-
-var util = require('./util');
+var Cookies = require('js-cookie');
 
 function setCookie(name, value, expiresAt) {
-  var expiresText = '';
-  if (expiresAt) {
-    expiresText = ' expires=' + util.isoToUTCString(expiresAt) + ';';
-  }
+  var cookieOptions = {
+    path: '/'
+  };
 
-  var cookieText = name + '=' + value + '; path=/;' + expiresText;
-  setCookie._setDocumentCookie(cookieText);
+  if (expiresAt && Date.parse(expiresAt)) {
```

Comment:
Two nits:
* We can omit the first condition (checking if `expiresAt` is truthy) as `Date.parse(...)` evaluates all falsy values to `NaN`
* Let's do `!!Date.parse(expiresAt)` instead so that it evaluates to `true`/`false` condition as opposed to `number`/`NaN`.

Therefore, `if (!!Date.parse(expiresAt)) {` seems to suffice.